tests/test_client.py

```diff
@@ -198,3 +199,70 @@ def test_client_credentials_grant(cc_grant_tuple):
     assert gc.auth_client.oauth2_validate_token(gc.garden_authorizer.access_token)[
         "active"
     ]
+
+
+def test_client_datacite_url_correct(
```

Comment:
dang, this is a beast of a unit test -- is this extra auth mocking because we couldn't instantiate the GardenClient without it?

Reply:
ok I thought this smelled familiar -- we do actually have an old `garden_client` fixture in `tests/conftest.py` that does a lot of similar setup -- if that fixture doesn't "just work" here (v possible, I think it predates some of the newer auth code in the client) let's update that fixture and ~unburden this test

Reply:
Yeah I needed it to construct the client. I was able to run it without all of the auth mocks locally, but it failed running via github actions.  I'll check out the fixture!